src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/API/APIApprovals.cs

```diff
@@ -22,9 +27,19 @@ public class ApiApprovals
         #endif
         public void ApproveAzureServiceBus()
         {           
-            var assembly = typeof(Message).Assembly;
+            var assembly = typeof(Message).Assembly;      
             var publicApi = Filter(PublicApiGenerator.ApiGenerator.GeneratePublicApi(assembly, whitelistedNamespacePrefixes: new[] { "Microsoft.Azure.ServiceBus." }));  
-            Approvals.Verify(publicApi);
+                                     
+            try
+            {
+              Approvals.Verify(publicApi);
+            }
+
```

Comment:
Our typical style guidance would be to remove the newline between the `try` and `finally` blocks.  However, if the existing convention in this library is to insert a newline, this is fine for now until we fix all the files to match our style guidelines.

Reply:
That just makes me sad.  Code density hurts readability, IMHO.

Reply:
I don't know if we have style guidelines specific to the Azure SDK yet, but in the meantime we should follow the .NET Core guidelines:

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

Reply:
Fixed.